src/main/kubectl/kubectl.ts

```diff
@@ -155,7 +155,7 @@ export class Kubectl {
       try {
         const args = [
           "version",
-          "--client", "true",
```

Comment:
This won't work because we support versions of kubectl going back to 1.10

Reply:
@Nokel81 did you look at the kubectl code for how options are handled? This change may be ok? it works on kube v1.16.15, though I don't know about 1.10. We may have been misinterpreting the kubectl docs on various options(?) Many have the default of `false`, which you get by not specifying the option. If you specify `--option` this means `true`.  Like for the set command example (https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#set):

```
kubectl set env pods --all --list
```
`all` and `list` are described the same way as `client` is for version. No need to specify `true` explicitly

Reply:
Oh okay. I guess I misunderstood the docs since I didn't look at the code.

Reply:
I tested with kubernetes 1.13.0 (oldest you can install with minikube) and it is fine. Looks like 1.24 is finickier so I think this PR is good and necessary moving forward.